packages/desktop-gui/src/project-nav/browsers_spec.jsx

```diff
@@ -59,7 +59,9 @@ describe('Browsers', () => {
     browserListOpened()
 
     cy.log('**closing**')
-    cy.get('.spacer').click()
+    // Click anywhere next to the dropdown to make sure it closes
+    cy.get('.spacer').click({ force: true })
```

Comment:
Without the force, the element was not visible as it's an "invisible" spacer - not sure why I had to adjust this as the previous test worked